script.js

```diff
@@ -61,11 +54,29 @@ function populateBookDetails(item) {
    const removeBtn = bkRow.appendChild(document.createElement('button'));
    removeBtn.innerHTML = 'Remove';
    removeBtn.setAttribute('class', 'removeBook')
```

Comment:
- [ ] Kindly make sure that when the remove button is clicked, the correct element is being deleted from your site and from the local storage :smile: